src/index.ejs

```diff
@@ -12,6 +12,16 @@
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <% if(__TARGET__ === 'mobile') { %>
   <script src="cordova.js" defer></script>
+  <% } else { %>
+  {{.CozyClientJS}}
```

Comment:
no, sorry if I did not explain very well: `cozy-client-js` is bundled in an unique `app.js` file with the other JavaScript code.